src/app/components/users-container/users-container.component.ts

```diff
@@ -13,16 +13,19 @@ export class UsersContainerComponent implements OnInit, OnDestroy {
   private subscriptions: Subscription = new Subscription();
   users: User[];
   isSubscriptionError: boolean;
+  isSearchComplete: boolean;
```

Comment:
Set default value for the variable. `false`

Reply:
okay.
it is initialized to default value in the latest commit.